test/NuGet.Clients.Tests/NuGet.VisualStudio.Common.Test/NuGetFeedbackDiagnosticFileProviderTests.cs

```diff
@@ -173,5 +184,147 @@ public async Task WriteToZipAsync_MefImportsNotAvailable_AddsMefErrorsFie()
                 Assert.Contains("mef-errors.txt", zipFiles);
             }
         }
+
+        [Fact]
+        public async Task WriteToZipAsync_WithMSSource_SourceRemainsStill()
+        {
+            // Arrange
+            var projectName = "testproj";
+            var dgSpecFileName = "dgspec.json";
+
+            using (var solutionManager = new TestSolutionManager())
+            {
+                var projectFolder = new DirectoryInfo(Path.Combine(solutionManager.SolutionDirectory, projectName));
+                projectFolder.Create();
+                var projectConfig = new FileInfo(Path.Combine(projectFolder.FullName, "project.json"));
+                var msbuildProjectPath = new FileInfo(Path.Combine(projectFolder.FullName, $"{projectName}.csproj"));
+                string extractPath = Path.Combine(solutionManager.SolutionDirectory);
+
+                BuildIntegrationTestUtility.CreateConfigJson(projectConfig.FullName);
```

Comment:
Is this ever cleaned up? I'm not seeing where it is, but I might be missing it. Test files should not remain after the test is finished.

Reply:
Yes, dispose in `TestSolutionManager` has built in clean up.